src/ck-core/ckarray.C

```diff
@@ -1813,6 +1813,13 @@ void CkArray::sendMsg(CkArrayMessage* msg, const CkArrayIndex& idx, CkDeliver_t
 // similar logic as sendMsg to forward the message, buffer it, or trigger demand creation.
 void CkArray::recvMsg(CkArrayMessage* msg, CmiUInt8 id, CkDeliver_t type, int opts)
 {
+  auto iter = locMgr->bufferedActiveRgetMsgs.find(id);
```

Comment:
This will be more encapsulated in CkLocMgr unless it turns out the buffering should be handled by CkArray instead.